src/graphql/utils/extendedFormatError.ts

```diff
@@ -0,0 +1,43 @@
+import { GraphQLError } from 'graphql/error'
+
+/**
+ * Given a GraphQLError, format it according to the rules described by the
+ * Response Format, Errors section of the GraphQL Specification, plus a few
+ * additional fields relevant to postgres errors, including HINT, DETAIL,
+ * and ERRCODE.
+ */
+export function extendedFormatError(error: GraphQLError, fields: Array<String>): GraphQLFormattedErrorExtended {
+  if (!error) {
+    throw new Error('Received null or undefined error.')
+  }
+  const originalError = error.originalError as GraphQLErrorExtended
+  fields = fields.map(field => field.toLowerCase())
```

Comment:
This should be done when the option is passed to postgraphql so it's only executed once in total rather than once for every error